android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotificationHelper.java

```diff
@@ -156,12 +156,33 @@ public void sendToNotificationCentre(Bundle bundle) {
                 ApplicationInfo appInfo = context.getApplicationInfo();
                 title = context.getPackageManager().getApplicationLabel(appInfo).toString();
             }
+            NotificationCompat priority;
+            if (bundle.getString("priority")) {
```

Comment:
does this work, do you not need to do a null check? 😮

Reply:
It seemed to work, but I'll test it once more to be sure!